packages/engine-server/src/topics/markdown/remark-abbr.d.ts

```diff
@@ -0,0 +1,14 @@
+declare module "remark-abbr" {
```

Comment:
I'm happy to get this into either Definitely Typed or directly upstream, if the maintainer is happy to accommodate the types.

Reply:
i think definitely typed is open to anyone contributing so you might just be able to submit a pull request directly

Reply:
Yup, I think so too.

I'll _try_ raising a PR against [zmarkdown](https://github.com/zestedesavoir/zmarkdown/tree/master/packages/remark-abbr) first, but since there are quite a few plugins there I suspect they'll be hesitant.